common/main/cpp/native_c4base.cc

```diff
@@ -36,7 +36,7 @@ using namespace litecore::jni;
  */
 JNIEXPORT void JNICALL Java_com_couchbase_lite_internal_core_C4Base_debug(JNIEnv *env, jclass ignore) {
     c4log_enableFatalExceptionBacktrace();
-    c4log_getWarnOnErrors();
+    c4log_warnOnErrors(true);
```

Comment:
I would caution that this will end up being noisy, and logging exceptions whenever they are thrown, even if they don't cause an actual issue (i.e .they are caught and handled later)

Reply:
Right.  Will take it out and put it back in 3.0, protected by the "DEBUGGING" flag.